### PR TITLE
v1 of html project

### PR DESCRIPTION
**Pull request by mayanktyagi45**
This is v1 of the html project

### CODE REVIEW

File: embedded.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title></title>
+        <link rel="stylesheet" href="./styles/embedded.css">
+    </head>
+    <body>
+        <div class="container">
+            <h1>Palolem Beach</h1>
+            <div class="sub-container">
+                <video width="400" controls>
+                    <source src="./assets/pexels_videos_2231485 (2160p).mp4" type="video/mp4">
```
Comment (PurnimaCompro) [inline embedded.html:14]: indententation should be with 2 spaces

File: embedded.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                    making it a favorite destination for both domestic and international tourists.</p>
+            </section>
+            <audio controls>
+                <source src="./assets/tropical-summer-music-112842.mp3" type="audio/mp3">
+            </audio>
+        </div>
+        
+        <script src="" async defer></script>
+    </body>
+</html>
```
Comment (PurnimaCompro) [inline embedded.html:30]: Remove Git error

File: index.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+            <td>Added new input types (e.g., date, email, number, etc.)</td>
+          </tr>
+        </table>
+      </div>
+
+      <!-- iframe -->
+
+    </main>
+
+    <iframe
```
Comment (PurnimaCompro) [inline index.html:97]: Remove extra line spaces

File: styles/embedded.css [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    padding: 0;
+}
+
+.container {
+    display: flex;
+    flex-wrap: wrap;
+    max-width: 800px;
+    margin: 0 auto;
+    padding: 20px;
+}
```
Comment (PurnimaCompro) [inline styles/embedded.css:13]: 2 spaces

File: styles/embedded.css [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+div > section {
+    flex-basis: 50%;
+    box-sizing: border-box;
+    padding: 10px;
+}
+
+audio {
+    width: 100%;
+    margin-top: 10px;
+}
```
Comment (PurnimaCompro) [inline styles/embedded.css:39]: Apply css on classes instead of selector

**Review by PurnimaCompro — CHANGES_REQUESTED** [review]

File: embedded.html (hunk 1 above)
Comment (mayanktyagi45) [reply embedded.html:14]: > indententation should be with 2 spaces

I used vs code extension prettier for indentation and it indented it this way.

Comment (mayanktyagi45) [reply embedded.html]: > indententation should be with 2 spaces

@PurnimaCompro I have made the suggested changes

**Comment by mayankcompro** [comment]
1. Understand branch naming convention. This naming convention not correct. 
2. Also go through some best practices regarding the commit message as well. For instance, we follow the commit guidelines as follows - 
  `feature-name: Add/Update/Implement/Rename/Delete/etc and then the details `
  
I hope this is clear if not we can meet and discuss.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+          <label class="label" for="email">Email:</label>
+          <input
+            class="input"
+            type="email"
+            id="email"
+            name="email"
+            placeholder="xyz@gmail.com"
+            required
+          />
+          <label class="label" for="lang">Language</label>
```
Comment (mayankcompro) [inline index.html:42]: Let's avoid using class name like `label` when we know that it is already applied on label tag. There are other instances as well like in input tag you are using the class as input. It should explain what this input is all about.

Improve all instances in this PR.

**Comment by mayankcompro** [comment]
Also, attach the screenshot of the UI that is designed by this HTML code in the PR description. So that the reviewer should get the proper understanding of the code.

**Review by mayankcompro — APPROVED** [review]

**Pull request merged**